src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -95,12 +97,10 @@ const OrderDetails: React.FC<Pick<Props, 'price' | 'buyToken' | 'sellToken' | 'p
   )
 }
 
-const UnfilledAmount: React.FC<Pick<Props, 'sellToken' | 'unfilledAmount' | 'unlimited' | 'pending'>> = ({
-  sellToken,
-  unfilledAmount,
-  unlimited,
-  pending,
-}) => {
+const UnfilledAmount: React.FC<Pick<Props, 'sellToken' | 'pending'> & {
```

Comment:
It is a bit hard to reson about Props in all these `Pick<Props, 'whatever'> & {}`
Maybe create interfaces `interface ** extends Pick<Props, 'whatever'> {`